### PR TITLE
Lantern Desktop Beta 8.0.0 collected testing issues

### PR DESCRIPTION
**Pull request by atavism**
Resolves https://github.com/getlantern/engineering/issues/1336

- Make text fields 56px high on desktop
- Remove floating label near cursor from text area on report issue screen

Screenshot 2024-04-28 at 9 51 20 PM

### CODE REVIEW

**Review by jigar-f — APPROVED** [review]
LGTM @atavism

File: lib/home.dart [hunk 1]
```diff
@@ -118,7 +118,7 @@ class _HomePageState extends State<HomePage> with TrayListener, WindowListener {
   void onTrayMenuItemClick(MenuItem menuItem) async {
     switch (menuItem.key) {
       case 'show':
-        windowManager.show();
+        windowManager.focus();
```
Comment (atavism) [inline lib/home.dart:121]: Thanks for making this change!

**Comment by atavism** [comment]
@jigar-f I made one more change to add an ffi method to exit the app (instead of calling `SystemChannels.platform.invokeMethod('SystemNavigator.pop')` that is called via Exit in the system tray menu. That seems to work much better. Do the changes still look good to you?

**Comment by jigar-f** [comment]
Let me check.

**Comment by jigar-f** [comment]
@atavism when exit is called by go, it's considered a crash on the flutter end. I see an error related to `Crashed Thread:        3  io.flutter.ui` I think we should use both, Go will end all go process and flutter will flutter process.

File: lib/ffi.dart [hunk 2]
```diff
@@ -21,7 +21,10 @@ void setLang(lang) => _bindings.setSelectLang(lang);
 
 String websocketAddr() => _bindings.websocketAddr().cast<Utf8>().toDartString();
 
-void ffiExit() => _bindings.exitApp();
+void ffiExit() {
+  _bindings.exitApp();
+  SystemChannels.platform.invokeMethod('SystemNavigator.pop');
```
Comment (jigar-f) [inline lib/ffi.dart:26]: @atavism I think we should use both, Go will end all go process and flutter will end flutter process

Comment (atavism) [reply lib/ffi.dart:26]: Sounds good. Thanks!

**Pull request merged**